lib/cdo/email_preference_constants.rb

```diff
@@ -0,0 +1,12 @@
+module EmailPreferenceConstants
+  SOURCE_TYPES = [
+    ACCOUNT_SIGN_UP = 'Teacher account sign up',
```

Comment:
I still think these don't need to be human-readable, since we'll need to loc them before we show them to users, we could use shorter strings to save space in the database, and if we're writing manual SQL queries against this data, having these human-readable strings seems more annoying than using shorter keywords that look more like variables.